extra-container/buildah/autobuild/build

```diff
@@ -0,0 +1,8 @@
+abinfo "Fetching source..."
+git clone https://github.com/containers/buildah
```

Comment:
Why not do this in `spec`?

Reply:
buildah will use git history during build, so a full repository is needed,

Reply:
> buildah will use git history during build, so a full repository is needed,

In this case, still use GITSRC + GITCO, then in prepare, invoke `acbs_copy_git`.